sdks/typescript/src/opik/client/Client.ts

```diff
@@ -0,0 +1,67 @@
+import { loadConfig, OpikConfig } from "@/config/Config";
+import { OpikApiClient } from "@/rest_api";
+import type { Trace as ITrace, ProjectPublic } from "@/rest_api/api";
+import { SavedTrace, Trace } from "@/tracer/Trace";
+import { v7 as uuid } from "uuid";
+
+interface TraceData extends Omit<ITrace, "startTime"> {
+  startTime?: Date;
+}
+
+export class OpikClient {
+  public apiClient: OpikApiClient;
+  public config: OpikConfig;
+  private existingProjects: Map<string, ProjectPublic> = new Map();
+
+  constructor(explicitConfig?: Partial<OpikConfig>) {
+    this.config = loadConfig(explicitConfig);
+
+    this.apiClient = new OpikApiClient({
+      apiKey: this.config.apiKey,
+      environment: this.config.host,
+      workspaceName: this.config.workspaceName,
+    });
+  }
+  public loadProject = async (projectName: string): Promise<ProjectPublic> => {
```

Comment:
i don't how much we want to align the typescript and the python sdks. i noticed that on the sdk side there is an approach of having `get_or_create_...` for such cases

Reply:
For now this is just internal use, I'll align more the things in the near future not at this stage